x/mint/abci.go

```diff
@@ -35,8 +35,11 @@ func BeginBlocker(ctx sdk.Context, k Keeper) {
 	}
 
 	communityPoolRatio := k.GetCommunityPoolRatio(ctx)
-	communityPoolCoins := k.GetCommunityPoolMint(ctx, communityPoolRatio, mintedCoin)
-	collectedFeesCoins := mintedCoins.Sub(communityPoolCoins)
+	communityPoolCoins := k.GetPoolMint(ctx, communityPoolRatio, mintedCoin)
```

Comment:
For future, need to make these a registration at app level.

Reply:
What do you mean? how can we do this at app level? the coins should be changing dynamically.

Reply:
I meant these should not be hardwired in the long run.